### DISCUSSION TITLE
alist-proxy

### DISCUSSION THREAD

**Discussion by iwoov** [AlistGo/alist#589] [Q&A] [answered]
alist-proxy -help描述这个太简单了，配置了好久都不能用。能否教下具体怎么用，举个例子就行

**Comment by xhofe** [comment, marked as answer]
最简单的使用：
```shell
./alist-proxy -host your_alist_adrress -token your_alist_token
```

**Reply by iwoov** [reply]
继续请教下，在自己的vps上运行以上命令后，填入的down_proxy_url是 http://my_ip:5243 这个吗

**Reply by xhofe** [reply]
最简单的情况是这样的，你也可以使用http://domain:5243 或者反向代理直接使用域名都可以，这和正常的网站是一样的。

**Reply by WXpiero** [reply]
我也在这里，卡住2天了，各种连不上，不论是直接写参数在js里还是写在命令里。。(;´༎ຶД༎ຶ`)

**Reply by Lau0x** [reply]
> 最简单的情况是这样的，你也可以使用http://domain:5243 或者反向代理直接使用域名都可以，这和正常的网站是一样的。

请教一下大佬，我是用A机器的Docker搭建的Alist，然后用B机器做了反向代理，访问alist的地址是`https://alist.com`我现在如果想要给OneDrive使用代理的方式观看视频，是不是只需要在OneDrive的后台配置的`down_proxy_url`这里填入`https://alist.com`就可以啦？还需要运行`./alist-proxy -host your_alist_adrress -token your_alist_token`这行代码么？

**Reply by xhofe** [reply]
你是想用哪个机器作为代理看视频？
如果是用A机器作为代理，则直接在账号设置中打开web_proxy/webdav_proxy就行了
如果要使用B机器作为代理，则需要在B机器上运行alist-proxy，然后把alist-proxy的地址填到down_proxy_url并打开web_proxy/webdav_proxy

**Reply by MynameisQingzhou** [reply]
大佬，我输完./alist-proxy -host your_alist_adrress -token your_alist_token这个指令下面提示listen and server :5243就不动了，怎么回事呀

**Comment by choateyang** [comment]
按大佬们的方法，确实成功了。
有点小问题，读取OD盘的视频，得代理那边全部缓存完毕才能正常播放。
如果使用cf workers，可以正常边看边拖动，不是缓存好才能播放视频。

**Comment by VOfficial404** [comment]
大佬，我这个怎么老是
{"code":500,"msg":"invalid character '\u003c' looking for beginning of value"}
